app/domain/processor.js

```diff
@@ -63,6 +65,8 @@ const processGrants = async ({ grants, manifestGrants, schemeName, containerClie
           sourceUrl: grant.url
         })
 
+        const shortSummary = await generateShortSummary(grant.content, summaryTokenLimit)
```

Comment:
Given `generateShortSummary` function can return an empty string, can we handle it. Either by not uploading a summary if the content is empty or have the function throw an error and handle it with a try and catch here